modules/web/src/app/shared/entity/rbac.ts

```diff
@@ -25,11 +25,21 @@ export class ClusterRole {
 }
 
 export class ClusterBinding {
```

Comment:
This class is exactly the same as `Binding` class below but if we want to keep both classes then I think we should rename `Binding` class to `NamespaceBinding` because we're using it in that context. Wdyt?